app/models/concerns/conclusion_reviews/gal_pdf.rb

```diff
@@ -238,6 +248,21 @@ def put_control_objective_items_table_on pdf, brief: false
       end
     end
 
+    def put_scope_detail_table_on pdf, brief: false
+      row_data = control_objectives_row_data brief, true
```

Comment:
```suggestion
    def put_scope_detail_table_on pdf
      row_data = control_objectives_row_data false, true
```
Si no lo usás lo sacaría.